assets/xml/objects/object_hidan_objects.xml

```diff
@@ -59,10 +59,11 @@
         <DList Name="gFireTempleSpinningFlamethrowerDL" Offset="0xAD00"/>
         <Collision Name="gFireTempleSpinningFlamethrowerCol" Offset="0xD5C0"/>
 
+                <DList Name="gFireTempleStationaryFlamethrowerShortDL" Offset="0xB0F8"/>
+        <Collision Name="gFireTempleStationaryFlamethrowerShortCol" Offset="0xCC90"/>
```

Comment:
any reason dlist is indented? if not can bring it back to line up with everything else